### PR TITLE
docs: chinese translations

### PR DESCRIPTION
**Pull request by enpitsuLin**
Working in progress reference #540 , but I also noticed some options `themeConfig` should be localized

***

<details>
<summary>Checklist for waiting translate page</summary>

### Guide

- [x] `/zh/guide/`
- [x] `/zh/guide/getting-started`
- [x] `/zh/guide/your-first-scene`
- [x] `/zh/guide/nuxt`
- [x] `/zh/guide/troubleshooting`
- [x] `/zh/guide/migration-guide`

### API

- [x] `/zh/api/tres-canvas`
- [x] `/zh/api/instances-arguments-and-props`
- [x] `/zh/api/compoables`
- [x] `/zh/api/events`

### Advanced

- [x] `/zh/advanced/extending`
- [x] `/zh/advanced/primitive`
- [x] `/zh/advanced/caveats`

### Debug

- [x] `/zh/debug/devtools`

### Examples

- [x] `/zh/examples/orbit-controls`
- [x] `/zh/examples/basic-animations`
- [x] `/zh/examples/groups`
- [x] `/zh/examples/load-textures`
- [x] `/zh/examples/load-models`
- [x] `/zh/examples/text-3d`
- [x] `/zh/examples/lights-shadows`
- [x] `/zh/examples/shaders`

### Directives

- [x] `/zh/directives/v-log`
- [x] `/zh/directives/v-light-helper`
- [x] `/zh/directives/v-always-look-at`
- [x] `/zh/directives/v-distance-to`

### Other

- [x] `/zh/team`

</details>

### CODE REVIEW

**Comment by enpitsuLin** [comment]
If any Chinese speaker user has some suggestions or wants to help me, comment or contact me pls.🥰

如果有任何中文使用者用户想给点建议或者帮忙的话，留下评论或者直接联系我就好🥰

**Comment by enpitsuLin** [comment]
I'm so happy to be working on this through collaborative translation with @X-Smart. 🤗

it will be done soon

**Comment by X-smart** [comment]
I'm glad to work together with [@enpitsuLin](https://github.com/enpitsuLin) on this translation.🤗

Hopefully, **Tres**  can help more people get their work done

**Comment by enpitsuLin** [comment]
@X-smart can you help review this pls? For my part 😊

**Comment by X-smart** [comment]
@enpitsuLin Of course, it would be my pleasure!

**Comment by X-smart** [comment]
@enpitsuLin 
如果您使用的是 Typescript，您就能充分享受类型的好处。 只需要保证你安装了 three 的类型定义。
-------
如果您使用的是 Typescript，并且安装了 three 的类型定义，您就能充分享受类型的好处。


要使模板编译器能与自定义渲染器一起工作，并且不会在控制台上发出警告，这样做是必须的。
------
这样做是为了避免在同时使用模板编译器能与自定义渲染器使控制台上发出警告。

沙盒 
------
Sandbox

动机
-----
起因

ThreeJS 是一个用于创建超棒 WebGL 3D 网站的奇妙的库
------
ThreeJS 在创建超棒 WebGL 3D 网站方面是一个奇妙的库
 
树摇的效果
------
摇树优化

由于 v2 是自定义的呈现器
------
由于v2是自定义渲染

在我们创建场景前，我们需要一个什么来展示它
-------
在创建场景前，我们需要一个什么来展示它？

 HTML 原始
-------
DOM节点

如果你的如果出现空白页
-------
如果出现空白页

那个场景看起来有点空
-------
上一步创建的场景看起来有点空

我们将传递一个 <TresTorusGeometry /> 和一个<TresMeshBasicMaterial />
------
我们添加 <TresTorusGeometry /> 和<TresMeshBasicMaterial />这两个组件。

Nuxt 所有附带的开发体验魔法
------
Nuxt附带的所有优秀开发体验

知道 3D 可以像缠结的毛线球
-----
知道 3D 就像缠结的毛线球

也可以像键盘上的猫
-----
或者像键盘上的猫

即使是最好的猫咪午睡者偶尔也会犯错误
-----
即使最好的猫咪偶有疏忽

我们的编码猫爱好者团队将采取行动
------
我们热爱编程的“猫托邦”团队将迅速出击

让我们一起让 TresJS 成为 Vue 中 做 3D 渲染最好的的猫咪！
-----
让我们携手合作，将 TresJS 打造成 Vue 3D 渲染界的璀璨明珠！

These are all my personal opinions, for reference only!

**Comment by alvarosabu** [comment]
Hi @enpitsuLin @hawk86104 thanks for taking care of this. We recently merged the Cookbook section #557 so we will need to update the PR.

Steps:

1. The `.md` files under `examples` directory are the same used in the cookbook, so just rename the directory from ~~cn/examples~~ to cn/cookbook.
2. Add a `cn/cookbook/index.md` like this https://github.com/Tresjs/tres/pull/558/files#diff-65c3e40a0b2bdcfae6c3be42b004bfa4e079a250fefed0fca56af95b33a209ca
3. Add the correspondent translated frontmatter meta to each one of the `.md` files inside the cookbook, like this https://github.com/Tresjs/tres/pull/558/files#diff-587026bcdc81388b1c6b391cdbdd7c2184a16b752a489c544a9d91c9e4b8f674
4. Update the config links https://github.com/Tresjs/tres/pull/558/files#diff-d9b1069ba7dab3215b86ba2b70b13a13977d3c09c11bdd8d468a0d7e8c686641

If you have any question or you need help with this, let me know 🙏🏻

**Comment by enpitsuLin** [comment]
hi @alvarosabu I'd kept up the new update already, could you pls takes some time review this. thx

BTW, the  preview of PR is so convenient😉

File: docs/.vitepress/config/zh.ts [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+              {
+                text: 'TresLeches 🍰',
+                link: 'https://tresleches.tresjs.org/',
+              },
+            ],
+          },
+        ],
+      },
+    ],
+    darkModeSwitchLabel: "外观",
```
Comment (alvarosabu) [inline docs/.vitepress/config/zh.ts:145]: Hi @enpitsuLin seems like `search` modal translations are missing 

Screenshot 2024-03-12 at 14 53 22

This is an example on how should be added https://github.com/Tresjs/tres/blob/b04e4aa3ba3390b58daf79454d8383c913715774/docs/.vitepress/config/de.ts#L145C5-L173C9

File: docs/.vitepress/config/zh.ts [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+          {
+            text: '演练场',
+            link: 'https://playground.tresjs.org/',
+          },
+          {
+            text: 'Github',
+            link: 'https://github.com/Tresjs/tres/',
+          },
+          {
+            text: 'Issues',
```
Comment (alvarosabu) [inline docs/.vitepress/config/zh.ts:122]: Is issues not translatable?

**Review by alvarosabu — CHANGES_REQUESTED** [review]
It looks amazing! I'm so happy to see this 💚, the Chinese community will love it.

@enpitsuLin there is only the search translation missing and I think we are good to go.

File: docs/.vitepress/config/zh.ts (hunk 1 above)
Comment (enpitsuLin) [reply docs/.vitepress/config/zh.ts:175]: updated

**Review by alvarosabu — APPROVED** [review]
Looks good to me @enpitsuLin 😍

**Comment by alvarosabu** [comment]
Hey @enpitsuLin there are some changes incoming from this PR #589, could you please apply them to the Chinese translation? With that we are ready to release this next week 💚

**Comment by enpitsuLin** [comment]
I'm sry that I was a little busy last weekend. I'll follow up in these days. @alvarosabu

**Comment by Sea-DH1** [comment]
@enpitsuLin 
primitive  用原语个人感觉不是很贴切
用  图元  会不会更好呢？

**Comment by enpitsuLin** [comment]
> @enpitsuLin primitive 用原语个人感觉不是很贴切 用 图元 会不会更好呢？

好像确实会更贴切点，不过这个 pr 已经合并了，不过你可以可以开另一个 pr 来修改下，可以 cc 我

**Comment by hexianWeb** [comment]
即将释出的 `tresjs version 4.0 `很酷 ! 我也希望能为社区出一份力 (●'◡'●)，如果有我可以帮忙的地方，请直接喊我

**Pull request merged**